install_homeassistant.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/sh
```

Comment:
There shouldn't need to be any change to this script since it's the one ran when we create the image. It's not run after the image is created so the tool shouldn't need to care about this script. Might move this script into the files folder but no change you need to do.